packages/react-form-state/docs/validators.md

```diff
@@ -35,6 +36,7 @@ function MyComponent() {
         description: 'Cool product',
         quantity: 0,
       }}
+      validateOnSubmit
```

Comment:
```suggestion
      validateOnSubmit={true}
```
or is just specifying the property magic syntax for the same thing?

Reply:
Yup it is for the same thing, just some syntactical sugar.